RdpGamepadPlugin/RdpGamepadPlugin.vcxproj

```diff
@@ -39,7 +39,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v142</PlatformToolset>
+    <PlatformToolset>v141</PlatformToolset>
```

Comment:
This would no longer build on a default installation of VS2019 without adding the 141 toolset package or having VS2017 installed as well.

Reply:
True, but getting it is a couple of clicks away for anyone using VS2019 (the installer lets you add it).  It's not available to install it for people using 2017.